cmd/agent/main.go

```diff
@@ -94,9 +94,14 @@ func setupZapLogger() zap.AtomicLevel {
 	}
 	cfg.EncoderConfig.EncodeTime = zapcore.RFC3339TimeEncoder
 	cfg.EncoderConfig.EncodeDuration = zapcore.StringDurationEncoder
+	ntpserv := os.Getenv("MA_NTP_SERVER")
+	if ntpserv == "" {
+		ntpserv = "pool.ntp.org"
+	}
```

Comment:
For consistency this check should be moved in `ensureDefaults()` in `config.go`.